docs/rst/tutorials/core/qos/mutable/mutable.rst

```diff
@@ -0,0 +1,296 @@
+=========================================
+
+.. contents::
+    :depth: 2
+    :local:
+    :backlinks: none
+
+Background
+----------
+
+Fast DDS over Vulcanexus offers the possibility of not only configuring the QoS policy when creating ROS nodes, but also to modify the mutable ones after the node/publisher/subscription creation.
+The QoS that allow their modification after that, are called mutable QoS.
+Please refer to `Fast DDS QoS Policy documentation <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/policy.html>`_ to get a list of all supported QoS and whether they are mutable.
+
+:ref:`tutorials_qos_ownership_ownership` explained the use of the Ownership and Ownership Strength QoS (see `Ownership QoS Policy <https://fast-dds.docs.eprosima.com/en/latest/fastdds/dds_layer/core/policy/standardQosPolicies.html#ownershipqospolicy>`_) and how to configure them within the ROS 2 talker/listener demo.
+
+This tutorial will show how to change Ownership Strength QoS in runtime, after all nodes have been already deployed.
+This will be done through the interaction of three nodes (one subscriber and two publishers):
+After creation, the subscriber will only be receiving data from the publisher with bigger ownership strength (corresponding to exclusive ownership QoS).
+After that, the ownership strength of the other one will be changed to become bigger, thus the subscriber will start to show the data of the latter.
+
+This will be done creating a custom package, following similar steps as in this :ref:`ROS 2 tutorial <MonitorParams>` to be able to change a node's parameter, and respond to that change by changing the Partition QoS of the publisher.
+
+The ROS 2 middleware layer (see :ref:`ROS 2 RMW <RO2Vendors>`) provides APIs to get handles to the objects of the inner DDS implementation, which is needed to be able to change the mutable Qos.
+Thus, this tutorial will also show how to use that powerful tool.
+For another demo on how to access inner RMW entities, see `demo_nodes_cpp_native <https://github.com/ros2/demos/tree/master/demo_nodes_cpp_native>`_.
+
+Prerequisites
+-------------
+
+The first prerequisite is to have Vulcanexus Humble installed (see :ref:`Linux binary installation <linux_binary_installation>` or :ref:`Linux installation from sources <linux_source_installation>`).
+
+Also, before starting this tutorial, user should be familiar with creating a workspace and creating a package, as well as familiar with parameters and their function in a ROS 2 system.
+The recommendation is to first complete the following tutorials:
+
+    * :ref:`Modifying Ownership and Ownership Strength QoS Policy <tutorials_qos_ownership_ownership>`
+
+    * :ref:`Writing a simple publisher and subscriber (C++) <CppPubSub>`
+
+    * :ref:`Understanding parameters <ROS2Params>`
+
+    * :ref:`Using parameters in a class (C++) <CppParamNode>`
+
+    * :ref:`Monitoring for parameter changes (C++) <MonitorParams>`
+
+Create a package
```

Comment:
I think we can skip this section and the next one if we do something like what is done [here](https://docs.vulcanexus.org/en/latest/rst/tutorials/core/deployment/dds2vulcanexus/dds2vulcanexus_topic.html#prerequisites). We put some commands so that the reader can copy-paste them and have de workspace fully set up. We don't need to explain how to modify CMakeLists.txt or package.xml either. That is explained in the other tutorials you've mentioned before. This way, the reader concentrates better on what we want to convey I think

Reply:
Done